server/fixed_stops.json

```diff
@@ -0,0 +1,62 @@
+{
+  "San Jose": [
```

Comment:
we only need this field right? the transit page only displays predictions from diridon

Reply:
this makes sense, for some reason i thought we just wanted to display all the ACE data (from every stop location)

resolved